app/models/concerns/hyrax/ability.rb

```diff
@@ -417,8 +417,10 @@ def can_review_submissions?
 
     def extract_subjects(subject)
       case subject
+      when Hyrax::WorkShowPresenter, FileSetPresenter
+        extract_subjects(subject.solr_document)
       when Draper::Decorator
-        super(subject.model)
+        extract_subjects(subject.model)
```

Comment:
skipping straight to `super` fails for nested decorators.

Reply:
I suspected that to be the case.